src/foundry/client/pixi/core/interaction/render-flags.d.mts

```diff
@@ -60,10 +68,12 @@ type _RenderFlags<Flags> = InexactPartial<{
 }>;
 
 declare global {
-  // @ts-expect-error - RenderFlag is built around willfully violating subclassing rules.
-  // The primary issue at hand is that each value can refer to the keys, thus making it technically
-  // unsound to apply a subclass to its superclass.
-  interface RenderFlag<out Flags> extends _RenderFlags<Flags> {}
+  /**
+   * @privateRemarks This *should* error but doesn't. RenderFlag is built around willfully violating subclassing rules.
```

Comment:
This no longer erroring makes sense because the `Flags` type parameter is completely unused. If we don't figure out a way to get the mixin to work then this comment should be removed.